adapter/config/default_config.go

```diff
@@ -58,6 +58,7 @@ var defaultConfig = &Config{
 			Location: "/home/wso2/security/truststore",
 		},
 		ArtifactsDirectory: "/home/wso2/artifacts",
+		SandboxVhost:       "vhost",
```

Comment:
That is correct, yes. For different gateways this can be different. Yes, the sandbox endpoint for an API in production comes in the API project `env_properties.json` file. 

As I understood, the prod-vhost and sand-vhost will always be different according to the requirement.